app/services/AddressLookupService.scala

```diff
@@ -40,16 +40,20 @@ class AddressLookupService @Inject() (
     signOutUrl: String,
     keepAliveUrl: String,
     lookupPageHeadingKey: String,
-    hintKey: String
+    hintKey: String,
+    editPageHeadingKey: String,
+    confirmationHeadingKey: String
   )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[AddressLookupOnRamp] =
     addressLookupConnector.initialiseJourney(
       AddressLookupRequest(
-        appConfig.selfUrl(callBackUrl),
+        appConfig.selfUrl(callBackUrl.split('?').head),
```

Comment:
AMBER: I didn't think we where manipulating the URL in the service anymore - we would take what the Controller passed us (so we could add additional parameters if necessary)

Reply:
I must have "prototyped" this fix on NDRC and then forgot to remove it.   Removed now.